stats/stats.go

```diff
@@ -348,6 +348,7 @@ func (e *HistogramData) Print(out io.Writer, msg string) {
 		if i == len(e.Data)-1 {
 			sep = "<=" // last interval is inclusive (of max value)
 		}
+		// nolint: gas
```

Comment:
what does the `gas` linter do?  and do we want it, considering we've added a ton of `// nolint: gas` statements all over this codebase?

Reply:
OK. Just seems to case more code harm than expected. Hopefully, it is catching useful things.